docs/Gamification.rst

```diff
@@ -0,0 +1,6 @@
+.. _Gamification:
```

Comment:
unrelated to space templates. should not be in this commit.

Reply:
I've started both documentations at the same time. My bad. i'll delete it.